helpers/manifest-parser.js

```diff
@@ -0,0 +1,329 @@
+/**
```

Comment:
move from helpers => parsers

I can see the need in the future for multiple parsers